### PR TITLE
Update augustus v.3.4.0

### PR DESCRIPTION
**Pull request by gallardoalba**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

This PR updates Augustus up to version 3.4.0. It includes the following modifications:
 - New supported species.
 - A new parameter has been included: softmasking

### CODE REVIEW

**Comment by gallardoalba** [comment]
Softmasking is enabled by default in this version.

**Comment by gallardoalba** [comment]
Could you have a look at this @abretaud? Let me know if you think it is necessary to include any additional options.

File: tools/augustus/augustus.xml [hunk 1]
```diff
@@ -205,23 +207,55 @@
...
+                    <option value="Fistulifera_solaris">Fistulifera solaris</option>
+                    <option value="Bathycoccus_prasinos">Bathycoccus prasinos</option>
+                    <option value="Chloropicon_primus">Chloropicon primus</option>
                 </param>
             </when>
         </conditional>
 
+        <param name="softmasking" type="boolean" argument="--softmasking"
+        label="Softmasking"
+        truevalue="--softmasking=1" falsevalue="--softmasking=0" checked="true"
```
Comment (abretaud) [inline tools/augustus/augustus.xml:255]: I'd move the `--softmasking=` part to the command block and keep only 1/0 here

**Review by abretaud — COMMENTED** [review]
LGTM thanks a lot for the update :)
Just a minor comment

File: tools/augustus/augustus.xml (hunk 1 above)
Comment (gallardoalba) [reply tools/augustus/augustus.xml:255]: Sure

**Review by abretaud — APPROVED** [review]

**Pull request merged**